### PR TITLE
hotfix/isaslicer2 updates

### PR DESCRIPTION
**Pull request by djcomlab**
Updates to ISAslicer2 for advanced slicing.

### CODE REVIEW

File: config/phenomenal_tools2container.yaml [hunk 1]
```diff
@@ -325,7 +325,7 @@ assignment:
 - docker_image_override: mtblisa
   docker_owner_override: phnmnl
   docker_repo_override: container-registry.phenomenal-h2020.eu
-  docker_tag_override: v0.10.0_cv0.7.10.98
+  docker_tag_override: latest
```
Comment (ilveroluca) [inline config/phenomenal_tools2container.yaml]: We need to fix these container versions!

**Review by ilveroluca — CHANGES_REQUESTED** [review]

File: config/phenomenal_tools2container.yaml (hunk 1 above)
Comment (djcomlab) [reply config/phenomenal_tools2container.yaml]: @ilveroluca I hadn't yet asked for a review yet, did I!? ;)

Comment (ilveroluca) [reply config/phenomenal_tools2container.yaml]: oops :joy:

Comment (djcomlab) [reply config/phenomenal_tools2container.yaml]: I'll fix the version when I'm sure it's ready! (soon)

**Comment by djcomlab** [comment]
Closing in favour of #312 as this dragged repeat commits when I rebased it...

**Pull request closed without merging**